### PR TITLE
16/17 merchant invoice index/show

### LINKED ISSUE #16: Merchant Invoices Index
Merchant Invoices Index

As a merchant,
When I visit my merchant's invoices index (/merchants/merchant_id/invoices)
Then I see all of the invoices that include at least one of my merchant's items
And for each invoice I see its id
And each id links to the merchant invoice show page

### LINKED ISSUE #17: Merchant Invoice Show Page
Merchant Invoice Show Page

As a merchant
When I visit my merchant's invoice show page(/merchants/merchant_id/invoices/invoice_id)
Then I see information related to that invoice including:
- Invoice id
- Invoice status
- Invoice created_at date in the format "Monday, July 18, 2019"
- Customer first and last name

### PR DESCRIPTION
**Pull request by thayes87**
*User stories 16&17*
-create feature test for both stories
-create new merchant/invoices directory in views with index and show pages

### CODE REVIEW

**Review by lcole37 — APPROVED** [review]

File: app/controllers/merchant/invoices_controller.rb [hunk 1]
```diff
@@ -1,5 +1,11 @@
 class Merchant::InvoicesController < ApplicationController
 
   def index
+    @merchant = Merchant.find(params[:merchant_id])
+    @invoices = @merchant.invoices.all.uniq
```
Comment (Dominicod) [inline app/controllers/merchant/invoices_controller.rb:5]: You may want to turn this into a model method.

**Review by Dominicod — APPROVED** [review]
Left one comment!

File: app/controllers/merchant/invoices_controller.rb (hunk 1 above)
Comment (rebeckahendricks) [reply app/controllers/merchant/invoices_controller.rb:5]: Agreed. Definitely model logic

**Review by rebeckahendricks — APPROVED** [review]
This looks great!!! Good job!

**Pull request merged**